### DISCUSSION TITLE
Feature Request : Manual start & Transfer timer

### DISCUSSION THREAD

**Discussion by Jkh62** [jgyates/genmon#1174] [Ideas]
I manually run my Gen the 1st of every month manually and take a look at the outputs to see if everything is running normal. I run it for 30 minutes and typically look at the computer time or my watch and try to ball park 30 minutes then shut it down.

It would be a cool feature to have a counter / timer start when the "start generator & transfer" tab is selected.
Nothing fancy , just a counter / timer for elapsed time since the start tab was started.

EXCELLENT software and really gives me peace of mind watching it cycle every month and knowing my gen and transfer switch is working correctly before an emergency takes place then finding out.

**Comment by jgyates** [comment]
Do you still want to manually start the cycle?  If not you can use the Enhanced Exercise Add On.  With the add on you can set it up to start at a specific time and date and it will run for a given number of minutes. 

If you want to manually start the cycle, I can add this to the [feature list](https://github.com/jgyates/genmon/blob/master/feature_requests.md).

Also, what type of controller do you have, Evo1, Ev2?

**Reply by Jkh62** [reply]
Manual start.
I really like the manual start and transfer option because it's let's me watch function in real time when I initate it.

My Gen is an Evo 2 controller.
I know the graph shows run time on the page with visual gauges but it's small and a simple elapsed time counter is pretty clear and simple. Not sure if it's simple to code.
